### ISSUE TITLE
Assault on Dragon Keep Support

### ISSUE THREAD

**Issue by apple1417** [BLCM/cabinetsorter#1]
Since AoDK standalone can be modded in the exact same way as BL2/TPS, we should add support for it in the BLCMods repo and thus also in Modcabinet.

**Comment by apocalyptech** [comment]
Yeah, agreed, that's been in my mind too.  I'd like to get a BLCMM update pushed out before we do so, though.  I was definitely planning on adding it.  :)
